### PR TITLE
Support details report

### PR DESCRIPTION
**Pull request by stevegrunwell**
Following up on further discussion w/ @chrislema re: #32, he was looking for more of a "easily provide the user with any details they might need to provide to support", a la Easy Digital Downloads.

This PR extends the work done in #34 to include a plaintext report that a user can easily copy + paste into a support ticket, giving us verification of the Airstory requirements, as well as details about their environment, active plugins, theme, etc.

### CODE REVIEW

File: airstory.php
Comment (bswatson) [inline airstory.php:20]: I'm under the assumption this will be changed to 1.0.0 when it is released to the .org repo.

File: includes/tools.php
Comment (bswatson) [inline includes/tools.php]: This feels like it should be wrapped in an `sprintf` to make the translation easier in the long term.

File: includes/tools.php
Comment (bswatson) [inline includes/tools.php]: Would the inclusion of this as a derivative work change the license to GPL 2.0+?

**Review by bswatson — CHANGES_REQUESTED** [review]

File: includes/tools.php
Comment (stevegrunwell) [reply includes/tools.php]: That's a really good question — I rewrote a fair portion of it (>50%, so it's more an "inspired by"), and it tears out all of the EDD-specific logic, but the pattern and formatting is still pretty damn similar. I'd like to avoid changing to GPL, if at all possible though, as the Airstory + Liquid Web teams were already in agreement on MIT (and MIT can be used more freely without the viral nature of the GPL).

At what point do you think we cross from being unique work to infringing on Pippin and team, thus invoking the GPL?

File: airstory.php
Comment (stevegrunwell) [reply airstory.php:20]: Yeah, I typically start everything off as a 0.1.0 but with the stability and amount of testing in this I'm comfortable with a 1.0.0.

File: includes/tools.php
Comment (stevegrunwell) [reply includes/tools.php]: A `sprintf()` or, you know, actually using a gettext function? Because I totally missed the latter, but not sure I agree on the former (especially since, for instance, the URL may change depending on the language, especially for the WordPress.org repo). Using the plugin in Spanish? Might as well link to the Spanish version of the repo, right?

**Review by norcross — APPROVED** [review]

File: airstory.php
Comment (stevegrunwell) [reply airstory.php:20]: #37 has been opened and the milestone updated to make sure this happens for release.

**Review by bswatson — APPROVED** [review]

File: includes/tools.php
Comment (stevegrunwell) [reply includes/tools.php]: Reached out to @pippinsplugins via Twitter DM to get his input, since he's the original copyright holder as owner of @easydigitaldownloads. I'd feel more comfortable getting his blessing before merging this + keeping MIT.

Comment (pippinsplugins) [reply includes/tools.php]: I have no issue with this. :+1: from me.

Comment (stevegrunwell) [reply includes/tools.php]: @pippinsplugins Thank you! Will note the link to this thread in the DocBlock for future reference.

Phil Dunphy (of Modern Family) indicating that you're awesome

**Pull request merged**
